consumer/consumererror/permanenterror_test.go

```diff
@@ -27,9 +28,12 @@ func TestPermanent(t *testing.T) {
 
 	err = Permanent(err)
 	require.True(t, IsPermanent(err))
+
+	err = fmt.Errorf("%w", err)
+	require.True(t, IsPermanent(err))
 }
 
 func TestIsPermanent_NilError(t *testing.T) {
 	var err error
 	require.False(t, IsPermanent(err))
-}
+}
```

Comment:
The lint task is failing due to missing a new line here
```suggestion
}

```